customize_image.sh

```diff
@@ -324,6 +324,14 @@ fi
 # peg cpu at 1200 MHz to maximize spi0 throughput and avoid jitter
 chroot "$IMAGEDIR" /usr/bin/revpi-config enable perf-governor
 
+# Since Raspberry Pi OS Bullseye the default user pi will only be used for the first
+# boot and then replaced by a username which has to be defined in the first boot wizzard.
+# Therefore we need to disable the userconfig and set the password to the previous default `raspberry`
+if [[ ! $(grep -q -E '^pi:\*' "$IMAGEDIR/etc/shadow" ]]; then
+	echo 'pi:$6$5usxnY6BwGl83hNQ$0GVkJryIjIXghLatO.wPACaDpg5h.wuZLxZr/Cw1Wx5gNZBADY6lbCRTI2J4TGR6/OdeW1trX3/PjmZtSjZhd/' | chroot "$IMAGEDIR" /usr/sbin/chpasswd -e
+	chroot "$IMAGEDIR" systemctl disable userconfig
+fi
+
```

Comment:
FYI, `echo pi:raspberry |  chroot "$IMAGEDIR"` /usr/sbin/chpasswd should work either, but not sure whether is is better or not. 
BTW, `wizzard ` should maybe `wizard`.

Reply:
wizzard: Well spotted, will update the comment and commit message. 
chpasswd: Makes sense and improves the readability of this line. Will update this a well.